README.md

```diff
@@ -1 +1,17 @@
-# -Hello-Microverse-
\ No newline at end of file
+# Hello-Microverse
```

Comment:
- [ ] [OPTIONAL] The Branch name should be the name of a feature or action, kindly in future projects consider using a meaningful name for your branch